.reuse/dep5

```diff
@@ -1,5 +1,5 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
-Files: .github/* nix/* protocols.json meta.json
+Files: .github/* nix/* protocols.json meta.json docker/package/defaults/* docker/package/scripts/defaults docker  /package/systemd/*
```

Comment:
Why did you put it here rather than as comments to the files? AFAIK comments are preferable. And AFAIU those files support comments.

Reply:
because I was lazy and wanted to make CI happy, will update it to the comments

Reply:
Btw, just found another bad thing here. `nix/*` is pretty big, it should be `nix/nix/*` here after recent restructuring. Could you please update it (unrelated to this PR, but a really small change)?